jumpscale/tools/servicemanager/servicemanager.py

```diff
@@ -0,0 +1,90 @@
+import gevent
+from signal import SIGTERM, SIGKILL
+
+from .services import StellarService, DiskCheckService
+
+from jumpscale.loader import j
+from jumpscale.core.base import Base
+
+DEFAULT_SERVICES = {"stellar": StellarService(), "disk-check": DiskCheckService()}
+
+# TODO: add support for non-periodic tasks
+# TODO: save services in config manager
+
+
+class ServiceManager(Base):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.services = DEFAULT_SERVICES.copy()
```

Comment:
I prefer we start them manually instead of having them configured in service manager, but let's see